README.md

```diff
@@ -9,9 +9,8 @@ that are useful to Ziti Tunnelers.
 
 ## What's a Ziti Tunneler?
 
-The main article about tunnellers is
-https://openziti.github.io/ziti/clients/tunneler.html. Editors may follow the
-"Improve this doc" link on any page.
+[The main article about tunnellers is here](https://docs.openziti.io/docs/reference/tunnelers/linux/). Editors may follow the
```

Comment:
> Editors may follow the "Edit this page" link on every page.

probably not entirely entirely accurate. that's still a wip on the docs stuff but... /shrug